dashboard/app/views/layouts/_javascript_strings.haml

```diff
@@ -91,6 +91,17 @@
         icon_library: {
           search_placeholder: "#{escape_t('components.icon_library.search_placeholder_text')}",
           no_icons_found: "#{escape_t('components.icon_library.no_icons_found')}",
+        },
+        playzone: {
+          artist_description: "#{escape_t('components.playzone.artist_description')}",
+          artist_title: "#{escape_t('components.playzone.artist_title')}",
+          continue_button: "#{escape_t('components.playzone.continue_button')}",
+          playlab_description: "#{escape_t('components.playzone.playlab_description')}",
+          playlab_title: "#{escape_t('components.playzone.playlab_title')}",
+          primary_header: "#{escape_t('components.playzone.primary_header')}",
+          projects_description: "#{escape_t('components.playzone.projects_description')}",
+          projects_title: "#{escape_t('components.playzone.projects_title')}",
+          secondary_header: "#{escape_t('components.playzone.secondary_header')}",
```

Comment:
I guess that in a world where code-studio and apps were part of the same pipeline, we could use the apps i18n instead of dashboard i18n here. Hopefully we get to that world soon :)